gc/gc.go

```diff
@@ -172,17 +172,26 @@ func ColoredSet(ctx context.Context, pn pin.Pinner, ng ipld.NodeGetter, bestEffo
 	// disk backed to conserve memory.
 	errors := false
 	gcs := cid.NewSet()
-	getLinks := func(ctx context.Context, cid cid.Cid) ([]*ipld.Link, error) {
-		links, err := ipld.GetLinks(ctx, ng, cid)
+	getLinks := func(ctx context.Context, CID cid.Cid) ([]*ipld.Link, error) {
```

Comment:
Please revert the variable name change.

Reply:
If I use cid as variable, variable is shadowing the cid package. 
I'm not able to use cid.Raw

Reply:
Ah, then I'd use `id, `c` or `key`.  Upper-case is usually reserved for exported variables.